cpp/lib/answer/json.cpp

```diff
@@ -321,6 +327,56 @@ JSON JSONBuilder::buildArray(const nlohmann::json& from) {
     return JSON(std::move(a));
 }
 
+void JSON::appendToString(std::stringstream& ss, const JSON& json) const {
```

Comment:
What's the reasoning behind not using `this` here?

Reply:
Alternatively:
```suggestion
void appendToString(std::stringstream& ss, const JSON& json) {
```
And remove from .hpp